scrappers/spiders/ELCIUDADANO_spider.py

```diff
@@ -0,0 +1,132 @@
+import logging
+
+import scrapy
+from scrapy.http.response.html import HtmlResponse
+
+logging.basicConfig(filename="run.log", encoding="utf-8", level=logging.DEBUG)
+
+
+def get_title(response: HtmlResponse) -> str:
+    return response.xpath('//*[contains(@class, "mb-4 the_title")]/text()').get()
+
+
+def get_epigraph(response: HtmlResponse) -> str:
+    return response.xpath('//*[contains(@class, "epigrafe mb-2")]/text()').get()
+
+
+def get_date(response: HtmlResponse) -> str:
+    try:
+        date = (
+            response.xpath('//span[contains(@class, "time-ago time-now")]')
+            .get()
+            .replace('<span class="time-ago time-now" data-date="', "")
+            .replace('" title=""></span>', "")
+        )
+        return date
+    except Exception as e:
+        logging.error(f"It was not possible to obtain the date. Error: {e}")
+        return None
+
+
+def get_authors(response: HtmlResponse) -> list[str]:
+    return response.xpath(
+        '//p[contains(@class, "mb-0 text-muted")]//a//text()'
+    ).getall()
+
+
+def get_authors_links(response: HtmlResponse) -> list[str]:
+    return response.xpath('//*[contains(@class, "mb-0 text-muted")]//@href').get()
+
+
+def get_comments(response: HtmlResponse) -> list[dict[str, str]]:
+    return [None]
+
+
+def get_excerpt(response: HtmlResponse) -> str:
+    return response.xpath('//*[contains(@class, "the-excerpt-")]//text()').get()
+
+
+def get_content(response: HtmlResponse) -> str:
+
+    content = " ".join(
+        response.xpath(
+            '//*[contains(@class, "the-content p402_premium-")]' "//p//text()"
+        ).getall()
+    )
+
+    titles = response.xpath(
+        '//*[contains(@class, "the-content p402_premium-")]//'
+        "*[self::h3 or self::h4 or self::h5 or self::h6]"
+        "//text()"
+    ).getall()
+
+    for title in titles:
+        content = content.replace(title, title + ".")
+
+    return content
```

Comment:
Te parece si agregas `clean_string` y `remove_social_networks` del módulo `utils` para hacer una limpieza del texto en este mismo lugar?
```suggestion
    content = remove_social_networks(response, content)
    cleaned_content = clean_string(content, ["string x reemplazar 1", "string x reemplazar 2", ...])
    return cleaned_content
```